### ISSUE TITLE
British English ‘unpublished’?

### ISSUE THREAD

**Issue by ZanderBrown** [microsoft/vscode-loc#443]
What happened?

**Comment by rswift** [comment]
I've been pondering the same thing... it seems to have vanished? VSCode throws a wobbler on load, and the language pack is no longer in the market place?

> The 1 extension(s) below, in workspace recommendations have issues: ms-ceintl.vscode-language-pack-en-gb (not found in marketplace)

**Comment by danyeh** [comment]
@Tyler James Leonhardt, no longer in microsoft publish scope?

**Comment by TylerLeonhardt** [comment]
👋  hi there. The English (UK) language pack is causing more harm than good now because it hadn’t been updated in a very long time… because of this, old strings are being used when new ones should so you run into situations where:
* the user sees placeholders (literally the text `{0}`) in the strings
* the user sees old text that has been updated

Because of this, and because another dialect of English is already supported by the product, we unlisted it for now until we have cycles to set up the right automation to make it happen. Sorry for unpublishing it out of the blue but the usage was also incredibly low which made us feel like not may folks would be impacted by this.

**Comment by ZanderBrown** [comment]
I'd be interesting in helping out with updating it, but really if your ending up with old strings used when the source has been updated it sounds like a flaw in the localisation system — Though I can't say I ever noticed any placeholders

> `trash -> bin`

In GNOME we actually call it the Wastebasket, but arguably things like colour are more important (I first noticed the pack was gone when I got no results for "colour" in the palette), there is also more to it than simply find/replace a load of -ize — There are grammatical differences as well. But of course, USian strings are a useful starting point.

(also it feels a bit odd you've entirely replaced the initial post, I'll agree the original wasn't entirely helpful but now I sorta read like an entitled idiot)

**Comment by rswift** [comment]
@ZanderBrown don't worry old bean, us entitled idiots can stick together and grumble about the lack of ColourfulBinLocalisations ;)

I'd also be willing to help out. The lack of usage does make me wonder if non-`en-US` speakers have become used to the US English content that people no longer bother seeking out localisation?

**Comment by thany** [comment]
I'm not even British myself and I want it back! I just like the UK variety of English more than the US one. It feels closer to home (and geographically speaking, it is).

As a Dutch boy, I'm very used to typing colour, metre (or meter if it refers to a device like a multimeter), customise and maybe Dockerise, and (less relevant) chips and crisps, and the boot and bonnet. So go ahead, mates, do feel a bit proud of your language 😎

**Comment by SergiusMath** [comment]
I agree with Zander, please try to return it.

**Comment by hickford** [comment]
Dates written in the American style are confusing to many people. Consider using a unambiguous date format such as yyyy-mm-dd (ISO 8601)

Example from Extension page

> Last updated 12/3/2021

![image](https://user-images.githubusercontent.com/105314/147394673-159f7479-cbe2-4b34-8859-e3d781b93090.png)

https://code.visualstudio.com/docs/getstarted/locales

**Comment by ZanderBrown** [comment]
Indeed, arguably even if en_GB is restored (one can dream) it would be more appropriate to use 8601 as it avoids confusion, and it is likely already familiar for programmers.

**Comment by nithin-mk** [comment]
As an Indian boy, I want the en-GB pack back too :-)

**Comment by cristianosuzuki77** [comment]
Internally tracking via ADO#1145337

**Comment by cristianosuzuki77** [comment]
Closing — en-GB is not currently a supported language for VS Code localization.

**Comment by ZanderBrown** [comment]
> Closing — en-GB is not currently a supported language for VS Code localization.

> Bring back British English language pack

🤔
